apps/web/src/components/features/properties/PropertyDetail.tsx

```
@@ -0,0 +1,323 @@
+  }
+  return property;
+};
+
+interface PropertyDetailProps {
+  id: string;
+}
+
+export const PropertyDetail = ({ id }: PropertyDetailProps) => {
+  // In a real app, this would use SWR or React Query with an API call
+  let property;
+  try {
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)